blueprints/frontend-shared-libs/index.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  name: 'frontend-shared-libs',
+    
+  afterInstall: function() {
```

Comment:
also I think it should be `return this.addBowerPackageToProject('node-uuid', '~1.4.3');`